integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/ui/portal/TransferFragment.kt

```diff
@@ -339,30 +339,51 @@ class TransferFragment : Fragment(R.layout.fragment_transfer) {
     }
 
     private suspend fun showWithdrawalLimitsError(period: WithdrawalLimitPeriod) {
-        val limits = viewModel.getWithdrawalLimits()
-        val okButtonText = if (period == WithdrawalLimitPeriod.PerTransaction) {
-            if (viewModel.onlineAccountStatus == OnlineAccountStatus.Done) {
-                getString(R.string.read_withdrawal_policy)
-            } else {
-                getString(R.string.online_account_create)
+        if (period == WithdrawalLimitPeriod.PerBlock) {
+            AdaptiveDialog.create(
+                R.drawable.ic_warning,
+                "Please wait before initiating the next withdrawal",
+                "Please wait 5 minutes before initiating another withdrawal",
+                getString(R.string.button_okay)
+            ).show(requireActivity()) {
+                // TODO: what do we do here?
+                // close this page?
+                // findNavController().popBackStack()
+                // show an error?
+                safeNavigate(
+                    TransferFragmentDirections.transferToResult(
+                        true,
+                        getString(R.string.crowdnode_withdraw_error),
+                        ""
+                    )
+                )
```

Comment:
To properly handle the progress dialog, we need to invoke the suspend method for showing the dialog which is called `showAsync` (not sure if it's a good name):
```suggestion
            ).showAsync(requireActivity())
```

Reply:
Fixed